modules/standard/IO.chpl

```diff
@@ -4781,6 +4781,12 @@ Syntax in Detail" section below.
 In this file, we use "integral" to refer to the Chapel types int or uint and
 "floating-point" to refer to real, imaginary, or complex, of any bit width.
 
+.. warning::
+
+   Binary conversions are now deprecated. Binary numeric conversions have been 
+   replaced by :proc:`channel.readBinary` and :proc:`channel.writeBinary`. 
```

Comment:
I think the issue that is triggering the docs failure is that these are ending up in a separate module from the functions - I think this attached to the submodule.  You should be able to resolve the problem by adding `IO.` or `~IO.` to the front of the reference